### PR TITLE
[risk=no][DB-831]update homepage Search bar and interaction

### PR DESCRIPTION
**Pull request by mhigbyflowers**

### CODE REVIEW

File: public-ui/src/app/data-browser/views/quick-search/quick-search.component.ts [hunk 1]
```diff
@@ -74,8 +74,6 @@ export class QuickSearchComponent implements OnInit, OnDestroy {
     localStorage.removeItem('ehrDomain');
     localStorage.removeItem('surveyModule');
     this.allOfUsUrl = environment.researchAllOfUsUrl;
-    this.dbc.getPmGroups().subscribe(results => {
```
Comment (mhigbyflowers) [inline public-ui/src/app/data-browser/views/quick-search/quick-search.component.ts:77]: I don't think this was doing anything

File: public-ui/src/app/data-browser/views/quick-search/quick-search.component.ts [hunk 2]
```diff
@@ -131,7 +135,6 @@ export class QuickSearchComponent implements OnInit, OnDestroy {
           if (!this.prevSearchText) {
             this.searchCallback(data);
           }
-          this.loading = false;
```
Comment (mhigbyflowers) [inline public-ui/src/app/data-browser/views/quick-search/quick-search.component.ts:134]: loading was set to false elsewhere.

File: public-ui/src/app/data-browser/views/quick-search/quick-search.component.ts [hunk 3]
```diff
@@ -207,7 +210,6 @@ export class QuickSearchComponent implements OnInit, OnDestroy {
         r.name, null, null);
     }
     localStorage.setItem('surveyModule', JSON.stringify(r));
-    localStorage.setItem('searchText', this.prevSearchText);
```
Comment (mhigbyflowers) [inline public-ui/src/app/data-browser/views/quick-search/quick-search.component.ts:210]: redundant

File: public-ui/src/app/data-browser/views/quick-search/quick-search.component.ts [hunk 4]
```diff
@@ -226,7 +228,6 @@ export class QuickSearchComponent implements OnInit, OnDestroy {
         r.name, null, null);
     }
     localStorage.setItem('ehrDomain', JSON.stringify(r));
-    localStorage.setItem('searchText', this.prevSearchText);
```
Comment (mhigbyflowers) [inline public-ui/src/app/data-browser/views/quick-search/quick-search.component.ts:229]: redundant

**Review by SrushtiGangireddy — APPROVED** [review]
Looks good. thanks

**Pull request merged**